### ISSUE TITLE
Some guidance on training?

### ISSUE THREAD

**Issue by yinhou** [monatis/german-tts#7]
Thanks for sharing your training results. I'm trying to reproduce the same process and perhaps do some finetuning afterwards. Now I have trained Tacotron2 94k steps, but still getting poor results.. I'm listing what I have done, would you mind sharing some guidance on what I have done wrong?

1. Cloned the TensorFlowTTS repo(after your PR merged)
2. Updated one line in tensorflow_tts/processor/thorsten.py to include special characters: ```_letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyzäüößÄÜÖ"```
3. Updated tensorflow_tts/configs/tacotron2.py to add support of 'thorsten' dataset: ```from tensorflow_tts.processor.thorsten import THORSTEN_SYMBOLS as thorsten_symbols ......        elif dataset == "thorsten":
            self.vocab_size = len(thorsten_symbols) ```
4. Updated examples/tacotron2/conf/tacotron2.v1.yaml to use dataset thorsten, and set batch_size to 9 for my 3070 gpu: ```dataset: thorsten ...... batch_size: 9```
5. Set up TensorflowGPU, did pip install, ran commands to preprocess the dataset:`tensorflow-tts-normalize --rootdir ./dump_thorsten --outdir ./dump_thorsten --config preprocess/thorsten_preprocess.yaml --dataset thorsten & tensorflow-tts-normalize --rootdir ./dump_thorsten --outdir ./dump_thorsten --config preprocess/thorsten_preprocess.yaml --dataset thorsten`
6. Trained Tacotron2 with command: `CUDA_VISIBLE_DEVICES=0 python examples/tacotron2/train_tacotron2.py   --train-dir ./dump_thorsten/train/   --dev-dir ./dump_thorsten/valid/   --outdir ./output/tacotron2/v1.thorsten/   --config ./examples/tacotron2/conf/tacotron2.v1.thorsten.yaml   --use-norm 1   --mixed_precision 0`
7. Verified result with the 'german-tts-inference' notebook you shared, using my 94k checkpoint h5 and your MB-MELGAN vocoder, the new vocab_size 70, and new mapper file. Here is the modified notebook with my 94k h5: [https://colab.research.google.com/drive/1VLjk3PouMlhAl0Tizko3OqH9CrNcvlzk?usp=sharing](url)

You can listen to the audio in the notebook, any help would be appreciated.
![tensorboard](https://user-images.githubusercontent.com/295752/108954598-037c6780-76a8-11eb-85bb-a5f798f72237.png)
![audio](https://user-images.githubusercontent.com/295752/108955325-edbb7200-76a8-11eb-971a-cb2014083d77.png)

**Comment by monatis** [comment]
hi @yinhou Thank you for your efforts to reproduce te results with the missing characters issue fixed.
You're setting `batch_size = 9`, so are you updating `gradient_accumulation_steps` in step 4 accordingly?

**Comment by yinhou** [comment]
Ah yes, that should be it. Thank you, @monatis ! I will use batch_size 8 and gradient_accumulation_steps 4 to try again.

**Comment by monatis** [comment]
@yinhou it may be a little bit slower but hopefully it helps. You can always ping me about it. Best of luck :-)
